src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -708,13 +725,33 @@ public void SetRigidity(int rigidity)
     }
 
     /// <summary>
-    ///   Removes organelles under the cursor
+    ///   Show options for the organelle under the cursor
     /// </summary>
     [RunOnKeyDown("e_secondary")]
-    public void RemoveOrganelle()
+    public void ShowOrganelleOptions()
     {
         GetMouseHex(out int q, out int r);
 
+        var organelle = editedMicrobeOrganelles.GetOrganelleAt(new Hex(q, r));
+
+        if (organelle == null)
+            return;
+
+        selectedHex = new Hex(q, r);
+
+        gui.ShowOrganelleMenu(organelle);
+    }
+
+    public void OrganelleInProcessOfMoving()
+    {
+        movingOrganelle = editedMicrobeOrganelles.GetOrganelleAt(selectedHex);
+    }
+
+    public void RemoveOrganelle()
+    {
+        int q = selectedHex.Q;
```

Comment:
and related to the previous one, this would maybe need to have the organelle passed in as well.

Though, then the symmetry based delete doesn't work.
Btw what happens if symmetry is on and the organelle menu is popup. What does it do in that case? I think you might be able to cheese the nucleus delete check in that case. So maybe the symmetry based delete checking should be removed entirely with this new popup menu.